### DISCUSSION TITLE
baresip/ sofia-sip?

### DISCUSSION THREAD

**Discussion by xpseudonym** [juha-h/baresip-studio#487] [Q&A] [answered]
Hi Folks,
I've recently started using `baresip` - when I did originally, about 13.06.2024, the user agent reported at my sip service provider was `baresip`.  Now the user agent reported is `sofia-sip`.  I guess it's kind'a cosmetic, but I'm intrigued by what might have caused the change in user agent reported by `baresip`?
Thank
Morgan.

**Comment by juha-h** [comment, marked as answer]
There has been no change in baresip app that could explain your issue.  Version 59.6.0 had this change:
```
- Added baresip, CPU, and Android information in User-Agent header
```
The header looks on my device like this:
```
User-Agent: baresip v60.4.0 (Android 14/aarch64)
```
There is no trace of `sofia-sip` anywhere.  Open the issue with your service provider.

**Reply by xpseudonym** [reply]
Wow!
Something weird is going on with my provider - it's now gone back to reporting `baresip` ...
That's in the time since my first post!
Thanks

**Comment by xpseudonym** [comment]
@juha-h I don't mean to be a complete pedant, but just checking that you'd agree that something pretty weird is going on when my service provide provides what's in the following pics depending on the time of day...:
![Screenshot_from_2024-07-01_15-41-01_munge](https://github.com/juha-h/baresip-studio/assets/5032475/12939e5b-5e65-4c3a-ace0-ed2dcbe13b78)
![Screenshot_from_2024-07-01_11-47-52_munge](https://github.com/juha-h/baresip-studio/assets/5032475/51865cd8-f826-441c-ac9d-fa5e9ebf3da6)
I'm about to take it up with them - but, I've asked them to fix some problems, might this be evidence of them doing some work?

Aha!  I worked it out, when I googled 'calls sofia-sip' and came up with this top of the list:
https://puri.sm/posts/voice-over-ip-in-gnome-calls-part-2-the-implementation/
So, although I've had the account disabled on my Librem 5 (which has been sitting on a charger doing nothing for months) it seems to have been registering all the same - how very buggy.  I turned the device off and it seems to have resolved the problem.  Sorry for all the noise...  (Obviously, Gnome Calls announces itself as 'calls sofia-sip')
M

**Comment by juha-h** [comment]
Good that you solved the issue.
